jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/AbstractAdvanceRuntimeDataServiceImpl.java

```diff
@@ -196,7 +196,6 @@ public void setEmf(EntityManagerFactory emf) {
                              "           FROM PeopleAssignments_PotOwners po \n" +
                              "           WHERE po.entity_id IN (:owners) \n" +
                              "           GROUP BY po.task_id \n" +
-                             "           HAVING COUNT(po.entity_id) = :num_owners \n" +
```

Comment:
Hi this is breaking AND operator and more likely to break backward compatibility.

Reply:
you need to add some sort of information to provide to the function whether it should be execute and AND or and OR

Reply:
How do you suggest to add this AND/OR option without modifying the existing API? Maybe create a new method?

Reply:
@ruromero  I think you can check for the presence  a "new" QueryParam in attributes args called, for example "CONDITION", which can take values "AND" or "OR", as done here https://github.com/kiegroup/jbpm/blob/master/jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/AbstractAdvanceRuntimeDataServiceImpl.java#L115

Reply:
Thanks for the suggestion @fjtirado I have added the queryParam but I have reused the MODE. Does it make sense?

Reply:
the right approach would be to transform this one 
 List<String> ownersArg
to 
QueryParam ownersArg
two new operandas for collections:
ANY (in the collection)
ALL (all in the collection)

reimplement current methods as QueryParam( list of owners)
ANY = OR
ALL = AND

Reply:
and add two new methods accepting the query param

Reply:
@elguardian let me know if this is what you had in mind and I'll proceed with the droolsjbpm-integration side